virl2_client/models/users.py

```diff
@@ -106,8 +107,9 @@ def create_user(
             "description": description,
             "admin": admin,
             "groups": groups or [],
-            "resource_pool": resource_pool,
         }
+        if self._is_rp_supported:
+            data["resource_pool"] = resource_pool
```

Comment:
so we assume the value is None then (as no RP could even be created with that server).